src/serviceorchestration/5_0_0/sd/service-orchestration-push-management/service_orchestration_push_management_sd.tex

```diff
@@ -0,0 +1,523 @@
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{2.8cm} | p{4.5cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Description \\ \hline
+id & \hyperref[sec:model:OrchestrationJobId]{OrchestrationJobId} & Unique job identifier. \\ \hline
+status & \pref{OrchestratioJobStatus} & Actual working state of the job. \\ \hline
+type & \pref{OrchestrationType} & Type of orchestration. \\ \hline
+requesterSystem & \pref{SystemName} & Name of the system that started the orchestration process. \\ \hline
+targetSystem & \pref{SystemName} & Name of the system for which the orchestration is executed. \\ \hline
+serviceDefinition & \pref{ServiceName} & Name of the service that the orchestration job is targeting. \\ \hline
+subscriptionId & \hyperref[sec:model:OrchestrationSubscriptionId]{OrchestrationSubscriptionId} & Unique identifier of associated subscription record. \\ \hline
+message & \pref{String} & Additional error or warning information. \\ \hline
+createdAt & \pref{DateTime} & The job was created at this timestamp. \\ \hline
+startedAt & \pref{DateTime} & The job was started at this timestamp. \\ \hline
+finishedAt & \pref{DateTime} & The job was finished at this timestamp. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{OrchestrationJobId}
+
+Unique \pref{String} identifier.
+
+\clearpage
+
+\msubsection{struct}{OrchestrationSubscriptionQueryRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | p{3.5cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+pageNumber & \pref{Number} & no (yes) & The number of the requested page. It is mandatory, if page size is specified. \\ \hline
+pageSize & \pref{Number} & no (yes) & The number of entries on the requested page. It is mandatory, if page number is specified. \\ \hline
+pageSortField & \pref{String} & no & The identifier of the field which must be used to sort the entries. \\ \hline
+pageDirection & \pref{Direction} & no & The direction of the sorting. \\ \hline
+ownerSystems &  \pref{List}$<$\pref{SystemName}$>$ & no & Requester is looking for subscriptions with any of the specified creator system names. \\ \hline
+targetSystems &  \pref{List}$<$\pref{SystemName}$>$ & no & Requester is looking for subscriptions with any of the specified target consumer system names. \\ \hline
+serviceDefinitions &  \pref{List}$<$\pref{ServiceName}$>$ & no & Requester is looking for subscriptions with any of the specified service definition names. \\ \hline
+\end{tabularx}
+\end{table}
+ 
+
+\clearpage
+
+\subsection{Primitives}
+\label{sec:model:primitives}
+
+Types and structures mentioned throughout this document that are assumed to be available to implementations of this service.
+The concrete interpretations of each of these types and structures must be provided by any IDD document claiming to implement this service.
```

Comment:
missing primitives (not mentioned the ones that are models but should be primitives):

- Boolean
- Direction
- Set